src/main/webapp/search-interview.js

```diff
@@ -23,6 +23,8 @@ function onSearchInterviewLoad() {
 function loadInterviews() {
   const searchResultsDiv = document.getElementById("search-results");
   searchResultsDiv.removeAttribute("hidden");
+  const position = document.getElementById('position').value;
+  // TODO: Include this position in the search process.
```

Comment:
Is this not implemented?

Reply:
Working on that now, wanted to get the frontend through first, can hold off and submit everything all together if that's preferred.